scripts/serve-backend.sh

```diff
@@ -0,0 +1,6 @@
+set -ex
+ROOT=$(realpath $(dirname $0)/..)
+
+npm install --prefix=$ROOT/app/server
+npm run build --prefix=$ROOT/app/server
+npm run serve --prefix=$ROOT/app/server
```

Comment:
This looks very much like the server part of `build.sh` - could these share code?

Reply:
i have abstracted out the build-backend script!